Blender.Client/batched_blender/batchlabs_request_handler.py

```diff
@@ -55,16 +55,19 @@ def _load_menu_options(self):
         try:
             response = urllib.request.urlopen(Constants.DATA_REPO_APP_INDEX_URL)
         except HTTPError as error:
-            self._logger.error("Failed to call the GitHub BatchLabs-data repository: " + str(error))
+            self._logger.error("Failed to call the GitHub BatchLabs-data "
+                               "repository: {}".format(str(error)))
```

Comment:
does this need a \ like your change in constants.py?

Reply:
:angel: Nopes :) `note: `  since this whole thins is in round brackets `()` this `\` is not needed.

Reply:
ahh, cool. thanks.